.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: ubuntu:latest
+    working_directory: /root/singnet/src/github.com/singnet/platform-pipeline
+    environment:
+      GOPATH: /root/singnet
+      SINGNET_REPOS: /root/singnet/src/github.com/singnet
+    steps:
+    - run:
+        name: Install tools
+        command: |
+          export PATH=$PATH:$GOPATH/bin
+          mkdir $GOPATH/download
+          cd $GOPATH/download
+          apt-get update
+          apt-get -y install sudo wget git
+          # Install NodeJS toolset
+          sudo apt-get -y install nodejs npm
+          # install Go tools
+          sudo apt-get -y install golang go-dep golang-goprotobuf-dev golint
+          # install IPFS
+          wget https://dist.ipfs.io/go-ipfs/v0.4.17/go-ipfs_v0.4.17_linux-amd64.tar.gz
+          tar xvfz go-ipfs_*.tar.gz
+          cp ./go-ipfs/ipfs /usr/local/bin
+          # Installl Python
+          sudo apt-get -y install python3 python3-pip
+          # Install other
+          sudo apt-get -y install libudev-dev libusb-1.0-0-dev
+    - run:
+        name: Build platform-contracts
+        command: |
+          cd $SINGNET_REPOS
+          git clone https://github.com/singnet/platform-contracts.git
+          cd platform-contracts
+          npm install
+          npm install ganache-cli
```

Comment:
Actually commands above doesn't build platform-contracts they just clone repository and install dependencies. To build contracts you should run: ```npm run-script compile``` and ```npm run-script package-npm```.